configuration/mysql/garage_port_drayage/README.md

```diff
@@ -0,0 +1,13 @@
+# C1T Garage Actions
+These are Port Drayage actions created the CDA1Tenth functionality and specifically tailored for the Turner Fairbank Highway Research Center's Saxton Laboratory garage demonstration.
+
+## Instructions
+Replace the port_drayage.sql file in docker-compose.yml with the files in this directory.
```

Comment:
I assume each of these is loaded independently for different behavior. I would suggest naming them more appropriately and maybe documented what happens in each one. If it is low effort to include MAP diagrams of what actions each result in it maybe worth while

Reply:
In XIL CARMA Cloud testing we did something similar in documenting the scenario we were loading. Take a look for reference.https://github.com/usdot-fhwa-stol/carma-config/tree/develop/xil_carma_cloud